packages/@aws-cdk/aws-redshift/README.md

```diff
@@ -15,4 +15,54 @@
 ---
 <!--END STABILITY BANNER-->
 
+### Starting a Redshift Cluster Database
+
+To set up a Redshift cluster, define a `Cluster`. It will be launched in a VPC.
+You can specify a VPC, otherwise one will be created. The nodes are always launched in private subnets and are encrypted by default.
+
+``` typescript
+import redshift = require('@aws-cdk/aws-redshift');
+...
+const cluster = new redshift.Cluster(this, 'Redshift', {
+    masterUser: {
+      masterUsername: 'admin',
+    },
+    vpc
+  });
+```
+By default, the master password will be generated and stored in AWS Secrets Manager.
+
+By default a default database (`default_db`) will be created upon construction. To change the name set `defaultDatabaseName` attribute.
```

Comment:
This sentence is a little unclear to me - a small re-wording might be in order perhaps?